packages/terra-clinical-onset-picker/src/terra-dev-site/doc/example/OnsetWithLegend.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import moment from 'moment';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import OnsetPicker from 'terra-clinical-onset-picker/lib/OnsetPicker';
+
+const OnsetWithLegend = () => (
+  <OnsetPicker
```

Comment:
Instead of adding a specific example with a legend we should update the existing examples to have legends. One of the reasons this issue was originally logged was that a fieldset without a legend is not accessible. In order for our site to be accessible all of the examples should have a legend.

Reply:
good call @jmsv6d !

Reply:
oh yes that's right will update the existing tests. thanks @jmsv6d